app/views/collections/show.rb

```diff
@@ -102,6 +103,21 @@ def version
 
     private
 
+    def og_image
```

Comment:
No, all three methods. All you need from `hero_config` is `:hero_image` which is `hero_image.file.present? ? hero_image.file.url : nil`, and therefore available to the `Page::Landing` model.